app/logic/daily_digest.rb

```diff
@@ -0,0 +1,32 @@
+class DailyDigest
+
+  def send
+    params = get_statistic
+    daily_statistic = DailyProgressDigest.new params
+
+    Notifier.digest(daily_statistic).deliver
+  end
+
+  private
+
+  def get_statistic
+    recipients = get_recipients
+    completed = completed_amount
+    remaining = incompleted_amount
+
+    Hash(recipients: recipients, completed: completed, remaining: remaining)
+  end
```

Comment:
@vlasiak, At that point, I would inline those 3 methods that determine recipients and amount of completed and remaining, but considering a future needs for another data, let's keep it as is.